src/app/modules/datagrid/datagrid-controller.js

```diff
@@ -143,13 +143,20 @@
 			$scope.getBooks();
 
 			var filterTags = function(searchTerm, cellValue, row, column) {
```

Comment:
If I'm reading it right, the effect of this change is that searchTerm is now a white-space-delimited list of search terms, each of which is an RE that must match at least one of the  values in cellValue (which rather surprisingly seems to be an array).
It's not obvious how this change is related to the purpose of this PR.
It's distinctly odd that the arguments row and column do not appear to be used.
I think at least this method needs a comment explaining what it does and perhaps how it is used (since I presume there is some special constraint on how it is called that explains the unused arguments).